blocks/header/header.css

```
@@ -1,260 +1,273 @@
 /* header and nav layout */
 header nav {
+  box-sizing: border-box;
   display: grid;
   grid-template:
     'hamburger brand tools' var(--nav-height)
-    'sections sections sections' 1fr / 50px 1fr 50px;
+    'sections sections sections' 1fr / auto 1fr auto;
   align-items: center;
-  padding: 0 2rem;
+  gap: 0 2em;
   position: fixed;
+  left: 50%;
+  transform: translateX(-50%);
   z-index: 1;
-  background-color: var(--background-color);
-  width: 100vw;
+  width: 100%;
+  max-width: 1200px;
```

Align with content max-width
```suggestion
  max-width: 1264px;
```

---

good catch! thank you!